pylab/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for pylab project.
```

Comment:
pylab is not the best name for a new project. 
It will clash with quite popular pylab library. 
https://scipy.github.io/old-wiki/pages/PyLab

Reply:
Renamed to myflights